### ISSUE TITLE
Widget Selection Issue

### ISSUE THREAD

**Issue by goyanipankaj** [odoo/odoo#33447]
Impacted versions: Odoo-12.0

Steps to reproduce: 
1) Create payments and try to select journal, But i am not able to see all the journals.
(Problem is widget selection).
Odoo Runbot link:
http://521272-12-0-39e05c.runbot15.odoo.com/web?#id=&action=485&model=account.payment&view_type=form&menu_id=502
Current behavior: Widget selection have not displaying all the data

Expected behavior: Show all the records

Video/Screenshot link (optional):
![Journal](https://user-images.githubusercontent.com/8874012/57863413-acc9d480-7817-11e9-887a-24bcae49b6a7.png)
![Payments](https://user-images.githubusercontent.com/8874012/57863418-b05d5b80-7817-11e9-8e6e-6ea3293d0a17.png)

**Comment by pedrobaeza** [comment]
That's because you have a very huge number of journals, which is not expected. Is that number real? :scream: 

The only solution in that case is to change through a view the widget type to regular one.

@ged-odoo any way, why selection widget is limited in the number of results?

**Comment by goyanipankaj** [comment]
Yes, I also want to know any limits on widget selection, same like tree view(default=80)?

**Comment by sswapnesh** [comment]
BTW, What is the benefit of using selection widget? (We can use options on m2o field to prevent additional operations on that)
